dartdoc_options.yaml

```diff
@@ -20,10 +20,13 @@ dartdoc:
     "Navigation View":
       markdown: doc/navigation-view.md
       name: Navigation View
+    "Map View":
+      markdown: doc/map-view.md
+      name: Map View
     "Image Registry":
       markdown: doc/image-registry.md
       name: Image Registry
-  categoryOrder: ["Navigation", "Navigation View", "Image Registry"]
+  categoryOrder: ["Map View", "Navigation", "Navigation View", "Image Registry"]
```

Comment:
nit: as this is navigation package we could move the "Map View" category after the "Navigation View"?

Reply:
My idea of putting it first was because the map itself is kind of most fundamental type and almost everything else is built on top of it (except nav session). But I can see your point, I'll move it.

Reply:
You have point there as well.
If you already moved it, let's not change anymore